src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,44 @@
+package racingcar;
+
+public class Car implements Comparable<Car> {
+
+    private static final int INIT_POSITION = 1;
+    private static final int LOWER_MOVABLE_BOUND = 4;
+
+    private int position = INIT_POSITION;
+    private String name;
+
+    Car() {
+    }
+
+    public Car(String name) {
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void attemptToMove(int number) {
+        if (isMovable(number)) {
+            position++;
+        }
+    }
+
+    private boolean isMovable(int number) {
+        return number >= LOWER_MOVABLE_BOUND;
+    }
+
+    @Override
+    public int compareTo(Car o) {
+        return position - o.position;
+    }
```

Comment:
```suggestion
    public int compareTo(Car other) {
        return position - other.position;
    }
```

마이너하지만 객체지향생활체조 6번 '축약하지 말라'를 참고해보시면 좋겠네요 🙂

 [객체지향생활체조](https://williamdurand.fr/2013/06/03/object-calisthenics/)

Reply:
네 앨런! 명심할께요! "축약하지 말라"!!!